README.md

```diff
@@ -210,15 +209,15 @@ Look for it to become `Ready`.
 
 #### Configure DNS
 
-For this tutorial a domain `on-k3s.dev` was purchased from Google Domains to show a full worked example.
+For this tutorial a domain `on-k3s.dev` was purchased from Google Domains to show a fully working example. If you are using a different domain registrar the following screens will look broadly similar.
```

Comment:
`worked example` is a term of phrase and the one I want to use here.

Reply:
OK

Reply:
Fine to keep your update though.